test/tst_encodeimg.jl

```diff
@@ -11,7 +11,7 @@ end
 
 @testset "encodeimg 256 small" begin
     @testset "gray square" begin
-        res, h, w = @inferred ImageInTerminal.encodeimg(ImageInTerminal.SmallBlocks(), ImageInTerminal.TermColor256(), gray_square, 2, 2)
+        res, h, w = @inferred ensurecolor(ImageInTerminal.encodeimg, ImageInTerminal.SmallBlocks(), ImageInTerminal.TermColor256(), gray_square, 2, 2)
```

Comment:
Won't this make `@inferred` test the returntype of `ensurecolor` instead of `encodeimg`